### PR TITLE
Public holidays

### PR DESCRIPTION
**Pull request by syedmoosa**
Added SDK for getting public holidays by country ID
[Core: Tech Task: Add Public Holidays in Moviebuff SDK #330](https://app.zenhub.com/workspaces/qube-slate-5b62c505eb87c774495f5301/issues/realimage/qube-slate/330)

### CODE REVIEW

File: moviebuff.go [hunk 1]
```diff
@@ -290,3 +292,41 @@ func (m *moviebuff) GetCertifications(country string) ([]Certification, error) {
...
+
+	calendarResp, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return nil, err
+	}
+
+	calendarInfo := new(Calendar)
+	err = json.Unmarshal(calendarResp, calendarInfo)
+	if err != nil {
+		log.Println(string(calendarResp))
```
Comment (svetha-cvl) [inline moviebuff.go]: There's no reason to log here, let's keep all logging on the client side. Clients can choose to use any loggers / log formats they want

File: moviebuff.go [hunk 2]
```diff
@@ -290,3 +292,41 @@ func (m *moviebuff) GetCertifications(country string) ([]Certification, error) {
...
+		return nil, err
+	}
+
+	res, err := new(http.Client).Do(r)
+	if err != nil {
+		return nil, err
+	}
+	defer res.Body.Close()
+
+	if res.StatusCode != http.StatusOK {
```
Comment (svetha-cvl) [inline moviebuff.go]: This block can be simplified by using a switch case

File: moviebuff.go (hunk 1 above)
Comment (syedmoosa) [reply moviebuff.go]: Removing it. Just logged for debugging. I forgot to remove it.

File: moviebuff.go (hunk 2 above)
Comment (syedmoosa) [reply moviebuff.go]: @svetha-cvl  Changed to switch case. Please review. @ilayaraja89

**Pull request merged**